site/gatsby-site/.eslintrc.json

```diff
@@ -16,6 +16,13 @@
       "alias": {
         "map": [
           ["@components", "./src/components"],
+          ["components", "./src/components"],
```

Comment:
What is the effect of listing/not listing here?

Reply:
it allows eslint to detect alias imports. otherwise, eslint will give module not found error on alias imports.